Cargo.toml

```diff
@@ -2,8 +2,16 @@
 name = "consensus"
 version = "0.1.0"
 edition = "2021"
+repository = "https://github.com/dusk-network/consensus"
+description = "An implementation of Succinct Attestation consensus protocol"
+categories = ["cryptography", "cryptography::cryptocurrencies"]
+keywords = ["consensus", "dusk", "cryptocurrency", "blockchain"]
+license = "MPL-2.0"
+exclude = [".github/*", ".gitignore", ".env", ".vscode"]
```

Comment:
I believe these are not required

> If include is not specified, then the following files will be excluded:
> 
> * If the package is not in a git repository, all "hidden" files starting with a dot will be skipped.
> * If the package is in a git repository, any files that are ignored by the [gitignore](https://git-scm.com/docs/gitignore) rules of the repository and global git configuration will be skipped.

_src here: https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields_

Reply:
`cargo publish --dry-run` fails if `exclude` does not list these files.

Reply:
Oh I see, that's because is a git repo, so the .gitignore rules